astropy/nddata/nddata_base.py

```diff
@@ -60,33 +57,13 @@ def wcs(self):
     @abstractproperty
     def meta(self):
         """
-        Metadata, if any, must be dict-like.
+        Meta information, if any.
```

Comment:
answer to both your comments: Since ``NDDataBase`` only defines the general interface for the container with meta informations I felt they were wrong there and moved them to ``NDData`` where that is actually enforced or at least recommended (because of the mixins). I can move them back or copy the docstrings of the ``NDData`` class into ``NDDataBase``.